PostDeploymentActions/updateMasterTimer.ps1

```diff
@@ -0,0 +1,6 @@
+$min = (Get-Date).Minute % 15 + 1
```

Comment:
I think we should remove `+1` here. If `$min` is 15 then I believe the cron executes only in 15, 30, 45 .

Reply:
this +1 was introduced to trigger master function within 1 min after deployment.
Another possible solutions could be:
```
$min = (Get-Date).Minute % 15
$new_schedule = "59 $min-59/15 * * * *"
```

Reply:
btw, I find this explanation very helpful (https://stackoverflow.com/questions/12786410/run-cron-job-every-n-minutes-plus-offset)

Reply:
I don't mind having 59 in seconds but I think it is just optimization we can not even rely on. However feel free to keep it but remove +1 please.

Reply:
Not sure why it is unreliable?
I think we should keep +1 min and use current seconds value as well to make sure the function is triggered within a minute after deployment is finished. I'll send an update shortly.

Reply:
@kkuzmin 
"Not sure why it is unreliable?" - Do you have any guarantee the deployment is finished within 59s after calling `(Get-Date)` ? 

I am against keeping +1 as it will run only 3x per hour if `$min` is 15. It is even clearly showed in the link you mentioned.